core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -562,6 +578,28 @@ public synchronized WorkerNetAddress getLocalWorker() throws IOException {
     return mLocalWorker;
   }
 
+  /**
+   * @return the info of all block workers eligible for reads and writes
```

Comment:
can you mention in javadoc the list of workers will be cached based on the expiration policy?

Reply:
Added! thanks